src/components/article/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import './viewArticle.scss';
+import PropTypes from 'prop-types';
+import {
+  TopComponent, LeftSideBar, ArticleContent, RecentArticles, RightSideBar, Footer,
+} from './viewArticleComponents';
+
+import { viewArticle } from '../../redux/actions/viewSingleArticle/viewSingleArticleAction';
+
+
+class ViewArticle extends Component {
+    state = {
+      getArticle: {
+        data: {
+          slug: '', title: '', description: '', body: '', readtime: '', createdAt: '',
+        },
+      },
+    }
+
+    componentDidMount() {
+      const { viewArticle: viewSingleArticle } = this.props;
+      viewSingleArticle('fakeslug2');
```

Comment:
Find a way of picking the slug from the url

Reply:
Fixed